test/test_jit.py

```diff
@@ -1693,6 +1693,17 @@ def constant_prop():
         self.run_pass('constant_propagation', constant_prop.graph)
         self.assertExpected(canonical(constant_prop.graph))
 
+    def test_trace_records_names(self):
+        def foo(bar, baz):
+            baz = bar + 3
+            quick_brown_fox = torch.neg(baz)
+            for i in range(20):
+                yeet = quick_brown_fox - 3.14
+            return yeet
+
+        traced = torch.jit.trace(foo, (torch.rand(3, 3), torch.rand(3, 3)))
+        self.assertExpectedGraph(traced.graph)
```

Comment:
Instead of piling on more expects that don't check much, can you simply check that `baz`, `bar`, `quick_brown_fox` and `yeet` appear in `str(traced.graph)`? That would be a muuuuch more robust test.